crates/ln-dlc-node/src/tests/system_tests/channel_less_payment.rs

```diff
@@ -0,0 +1,154 @@
+use std::str::FromStr;
+use std::time::Duration;
+// Related issue: https://github.com/get10101/10101/issues/55.
+use crate::tests::system_tests::create_tmp_dir;
+use crate::tests::system_tests::fund_and_mine;
+use crate::tests::system_tests::has_channel;
+use crate::tests::system_tests::init_tracing;
+use crate::tests::system_tests::log_channel_id;
+use crate::tests::system_tests::setup_ln_node;
+
+#[tokio::test]
+async fn given_no_channel_with_coordinator_when_invoice_generated_then_can_be_paid_through_coordinator(
+) {
+    init_tracing();
+
+    let test_dir = create_tmp_dir("multi_hop_test");
+
+    // 1. Set up three LN-DLC nodes.
+    let alice = setup_ln_node(&test_dir, "alice").await;
+    let coordinator = setup_ln_node(&test_dir, "coordinator").await;
+    let bob = setup_ln_node(&test_dir, "bob").await;
+
+    tracing::info!("alice: {}", alice.info);
+    tracing::info!("coordinator: {}", coordinator.info);
+    tracing::info!("bob: {}", bob.info);
+
+    let _alice_bg = alice.start().await.unwrap();
+    let _coordinator_bg = coordinator.start().await.unwrap();
+    let _bob_bg = bob.start().await.unwrap();
+
+    // 2. Connect the two nodes.
+
+    tokio::time::sleep(Duration::from_secs(2)).await;
+    alice.keep_connected(coordinator.info).await.unwrap();
+    bob.keep_connected(coordinator.info).await.unwrap();
+    alice.keep_connected(bob.info).await.unwrap();
+
+    // 3. Fund the Bitcoin wallets of the nodes who will open a channel.
+    {
+        bob.fund(bitcoin::Amount::from_sat(1_000_000))
+            .await
+            .unwrap();
+        coordinator
+            .fund(bitcoin::Amount::from_sat(1_000_000))
+            .await
+            .unwrap();
+
+        // we need to wait here for the wallet to sync properly
+        tokio::time::sleep(Duration::from_secs(5)).await;
+
+        alice.sync();
+        let balance = alice.wallet.inner().get_balance().unwrap();
+        tracing::info!(%balance, "Alice's wallet balance");
+
+        coordinator.sync();
+        let balance = coordinator.wallet.inner().get_balance().unwrap();
+        tracing::info!(%balance, "coordinator's wallet balance");
+
+        bob.sync();
+        let balance = bob.wallet.inner().get_balance().unwrap();
+        tracing::info!(%balance, "bob's wallet balance after calling faucet");
+    }
+
+    tracing::info!("Opening channel");
+
+    // 4. Create channel between bob and coordinator.
+    bob.open_channel(coordinator.info, 30000, 0).unwrap();
+
+    tokio::time::sleep(Duration::from_secs(2)).await;
+
+    // Add 1 confirmation required for the channel to get usable.
+    // we are mining to a random address to no pollute the users wallets
+    let address = Address::from_str("bcrt1qylgu6ffkp3p0m8tw8kp4tt2dmdh755f4r5dq7s")
+        .expect("To be a valid address");
+    fund_and_mine(address.clone(), bitcoin::Amount::from_sat(1000)).await;
+
+    // Add 5 confirmations for the channel to get announced.
+    for _ in 1..6 {
```

Comment:
❓ since we change the channel config to forward also on private channels, do we still need these confirmations?